sources/api_client.go

```diff
@@ -217,26 +218,34 @@ func (sc *SourcesClient) GetInternalAuthentication(authID string) (*model.Authen
 }
 
 func (sc *SourcesClient) headers() map[string][]string {
+	var headers = make(map[string][]string)
+
+	headers["Content-Type"] = []string{"application/json"}
+
 	if conf.SourcesPSK == "" {
-		var xrhid string
+		var xRhId string
+
 		if sc.IdentityHeader == "" {
-			xrhid = encodedIdentity(sc.AccountNumber)
+			xRhId = encodeIdentity(sc.AccountNumber, sc.OrgId)
 		} else {
-			xrhid = sc.IdentityHeader
+			xRhId = sc.IdentityHeader
 		}
 
-		return map[string][]string{
-			"x-rh-identity": {xrhid},
-			"Content-Type":  {"application/json"},
-		}
+		headers["x-rh-identity"] = []string{xRhId}
 	} else {
-		return map[string][]string{
-			"x-rh-sources-psk":            {conf.SourcesPSK},
-			"x-rh-sources-account-number": {sc.AccountNumber},
-			"Content-Type":                {"application/json"},
-			"x-rh-identity":               {sc.IdentityHeader},
+		headers["x-rh-sources-account-number"] = []string{sc.AccountNumber}
```

Comment:
We probably just need an if around this account-number header like the OrgID header since there is a chance it'll be `""`.

Reply:
You're right, yeah. Implemented!